cibuildwheel/__main__.py

```diff
@@ -185,7 +185,7 @@ def main() -> None:
         print('cibuildwheel: Could not find setup.py, setup.cfg or pyproject.toml at root of package', file=sys.stderr)
         exit(2)
 
-    if args.archs is not None:
+    if args.archs:
```

Comment:
I'm in favor of this change, though, since it lets `--archs=""` be ignored, which makes logic like that above simpler.

Reply:
This could be hiding configuration mistakes, though. `--archs=$TYPO_ENV_VAR` or  `--archs=${{ matrix.bad_conf_var }}` or so.

(Not sure what I think is best, but I think it's an important thing to note about this change.)

Reply:
Fair point. I could just do the extra bash logic to add the `--arch` if non-empty.

Reply:
Yeah, though it might be nice behavior to our users, as well, as you demonstrated a use case. I'm not sure what's optimal, here.

(Not to worried about making our one time use in the action slightly uglier, actually :-p )